src/deploy.cmd.js

```diff
@@ -209,6 +222,8 @@ class DeployCommand extends AbstractCommand {
     } catch (e) {
       this._pkgJson = {};
     }
+    this._updated = this._updated || new Date().getTime();
+    this._updatedBy = this._updatedBy || os.userInfo().username;
```

Comment:
I’d like us to find a more privacy-friendly default for that. Having your user name in an annotation feels sneaky and surprising, especially when it’s done without an opt-in.

Reply:
hmm..ok. I remove the updatedBy for now.